src/wccp2.cc

```diff
@@ -1640,12 +1642,19 @@ wccp2HereIam(void *)
                                 router,
                                 &service_list_ptr->wccp_packet,
                                 service_list_ptr->wccp_packet_size);
+                // XXX: setup fd_table[] properly so we can use FD_WRITE_METHOD()
+                fd_table[theWccp2Connection->fd].bytesWritten(service_list_ptr->wccp_packet_size);
```

Comment:
Same here: Missing protection from comm_udp_sendto() failures. GitHub will not let me post a specific correction for this one.